Formula/dmd.rb

```diff
@@ -59,6 +63,7 @@ def install
       ENABLE_RELEASE=1
     ]
 
+    inreplace "src/build.d", old_host_dmd_ver, host_dmd_ver
```

Comment:
I trust you this is really needed but could you explain why in a few words? Maybe an inline comment would help people who will need to update this formula in the future?

Reply:
Good point. Added

Also related Upstream PR: https://github.com/dlang/dmd/pull/10489